stable/memcached/README.md

```diff
@@ -0,0 +1,68 @@
+# Memcached
+
+> [Memcached](https://memcached.org/) is an in-memory key-value store for small chunks of arbitrary data (strings, objects) from results of database calls, API calls, or page rendering.
+
+This chart is based off of earlier work in the [Bitnami charts repo](https://github.com/bitnami/charts). 
```

Comment:
@gtaylor May I suggest changing it to: 

Based on the [memcached](https://github.com/bitnami/charts/tree/master/incubator/memcached) chart from the [Bitnami Charts](https://github.com/bitnami/charts) repository.

Thank you.